plugins/out_s3/s3.c

```diff
@@ -768,6 +768,18 @@ static int cb_s3_init(struct flb_output_instance *ins,
         ctx->storage_class = (char *) tmp;
     }
 
+    /* Formatting */
+    ctx->format = S3_FORMAT_JSON;
+    tmp = flb_output_get_property("format", ins);
```

Comment:
Looks like this new option is incompatible with our log key option, so that should be called out in the config map help text and guarded against here.

Reply:
@PettitWesley I just pushed a fix to address that